app/models/Organization.ts

```diff
@@ -0,0 +1,61 @@
+import { get } from '../utils/DataService';
+import { parseAPISchedule } from '../utils/transformSchedule';
+import { Service, shouldInheritSchedule } from './Service'; // eslint-disable-line import/no-cycle
```

Comment:
I'm actually used to this rule not throwing errors at all when importing only types circularly even without `import type {} from`, I added this ignore with the plan to go back and figure out why.

But you're right, this is gross. Moving the actual shared util functions to Meta and importing types only.

Reply:
I chose to keep Organization and Service here, because I figure they will have a lot of utility functions, I imagine "models" includes the interface, and model specific business logic we abstract out of components as we move to TS.

For example, a lot of functions inside the edit page should be generic abstractions moved into Organization and Service with their own unit tests.

Meta could be split out if it gets complex or has many helper functions, I just figured those models will never get as complex and splitting out 10 different files now would be premature.